File: Java-164/chapter.9/lab9.challenges/PhonePlan.java
// ===== Code from file PhonePlan.java =====
public class PhonePlan {
    private int freeMinutes;
    private int freeMessages;
 
    public PhonePlan() {
        freeMinutes  = 0;
        freeMessages = 0;
     }
 
     // FIXME: Create a second constructor with numMinutes and numMessages parameters.
    public PhonePlan(int numMinutes, int numMessages) {
       freeMinutes = numMinutes;
       freeMessages = numMessages;
    }
    /* Your solution goes here  */
 
    public void print() {
       System.out.println("Minutes: " + freeMinutes + ", Messages: " + freeMessages);
    }
 }

File: Python-164/chapter.9/lab9.challenges/phonePlan.py
class PhonePlan:
    freeMinutes = 0
    freeMessages = 0

    def __init__(self, numMinutes, numMessages):
        self.freeMinutes = numMinutes
        self.freeMessages = numMessages

    def print(self):
        print(f"Minutes: {self.freeMinutes}, Messages: {self.freeMessages}")